### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by arturo-seijas**

### CODE REVIEW

**Review by nrobinaubertin — DISMISSED** [review]

File: ca.pem
Comment (nrobinaubertin) [inline ca.pem:1]: Is this supposed to be here ? (in a doc PR ?)

File: docs/index.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Charmed Wazuh Dashboard Documentation
+The [Wazuh Dashboard](https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/index.html)
+is a component for analyzing and visualizing security data data based on [OpenSearch](http://opensearch.org/). Consequentily,
```
Comment (nrobinaubertin) [inline docs/index.md:3]: s/Consequentily/Consequently

File: docs/index.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+this charm is a Charmed OpenSearch Dashboards fork.
```
Comment (erinecon) [inline docs/index.md:4]: Nitpick: I would link the Charmed OpenSearch Dashboards repo here

File: docs/index.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
 
+For the complete documentation, refer to [upstream](https://charmhub.io/opensearch-dashboards).
```
Comment (erinecon) [inline docs/index.md:6]: You may also want to include a link to the Wazuh Server doc set

File: docs/index.md [hunk 4]
```diff
@@ -1,3 +1,17 @@
...
+## Project and community
+
+The Wazuh Dashboard Operator is a member of the Ubuntu family. It's an open-source project that warmly welcomes community projects, contributions, suggestions, fixes, and constructive feedback.
+
+- [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+- [Get support](https://discourse.charmhub.io/)
+- [Join our online chat](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
+- [Contribute](https://github.com/canonical/opensearch-operator/blob/2/edge/CONTRIBUTING.md)
+
+Thinking about using the Wazuh Server Operator for your next project? [Get in touch](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)!
```
Comment (erinecon) [inline docs/index.md:17]: Did you mean to say "Server" here, or should it say Dashboard?

Comment (arturo-seijas) [reply docs/index.md:17]: Fixed

File: docs/index.md (hunk 3 above)
Comment (arturo-seijas) [reply docs/index.md:6]: Where and how?

File: ca.pem
Comment (arturo-seijas) [reply ca.pem:1]: That's a leftover from some tests. Thx

**Review by nrobinaubertin — DISMISSED** [review]

File: docs/index.md (hunk 3 above)
Comment (arturo-seijas) [reply docs/index.md:9]: Updated

**Review by erinecon — APPROVED** [review]

**Review by amandahla — APPROVED** [review]

**Pull request merged**